python/lsst/rubintv/production/pipelineRunning.py

```diff
@@ -332,9 +337,17 @@ def postProcessCalibrate(self, quantum, processingId):
         # mechanism too, and anything else which forks off the main processing
         # trunk.
         self.watcher.redisHelper.reportFinished(self.instrument, "binnedCalexpCreation", processingId)
-
         self.log.info(f"Wrote binned calexp for {dRef.dataId}")
 
+        try:  # TODO: remove the try once this is known to work
```

Comment:
Ah, interesting. Yes and no. Yes, it's good and can be removed for the summit, but the try is needed because it should and does fail when this runs at USDF for NV processing, because that's not supposed to insert to consDB (or if it does, it needs to use a different table). So, I'll change the comment, and note why it's needed and what the trajectory for getting that sorted more properly is with a ticket.

Reply:
DM-45438